### ISSUE TITLE
Ignore Markdown "quoted" text

### ISSUE THREAD

**Issue by phillipjohnston** [vale-cli/vale#102]
When linting documentation, I notice that quotes (paragraphs preceded by `>`) are not ignored. For instance:

> Mature frameworks usually incorporate several design patterns. The patterns help make the framework’s architecture suitable to many different applications without redesign. An added benefit comes when the framework is documented with the design patterns it uses [BJ94]. People who know the patterns gain insight into the framework faster. Even people who don’t know the patterns can benefit from the structure they lend to the framework’s documentation. Enhancing documentation is important for all types of software, but it’s particularly important for frameworks. Frameworks often pose a steep learning curve that must be overcome before they’re useful. While design patterns might not flatten the learning curve entirely, they can make it less steep by making key elements of the framework’s design more explicit.

Will yield (with my rules):

```
 46:659  warning  Craft sentences at 25 words or  documentation.SentenceLength
                  fewer, whenever possible.
 46:766  warning  Avoid using 'key' (unless it    documentation.Clarity
                  unlocks something).
```

I don't actually want to address these warnings since it would involve modifying a third-party quote.

**Comment by phillipjohnston** [comment]
(Perhaps there is a way to configure this behavior locally, but I did not find that option after poking around the documentation)

**Comment by jdkato** [comment]
I added the ability to customize what scopes Vale will skip in https://github.com/errata-ai/vale/commit/95601cddc4dc072fe179e3f0bba2f8c0530023b1 and it will be available in the next release:

```ini
MinAlertLevel = suggestion

# `script`, `style`, `pre`, and `figure` are the defaults.
SkippedScopes = script, style, pre, figure, blockquote

[*]
...
```

In the meantime, you can always disable Vale using comments as mentioned in https://github.com/errata-ai/vale/issues/101#issuecomment-454564822.

**Comment by phillipjohnston** [comment]
Great, thank you!

**Comment by jdkato** [comment]
I've released `v1.3.0`.
